docs/CORTX-S3 Server Quick Start Guide.md

```diff
@@ -163,7 +153,7 @@ Before your test your build, ensure that you have installed and configured the f
 2. Ensure you've installed pip.
     - To check if you have pip installed, run the command: `$ pip --version`
     - To install pip, run the command: `$ easy_install pip`
-3. If you don't have Python Version 2.6.5+, install Python using: `$ yum install python26`
+3. If you don't have Python Version 2.6.5+, install Python using: `$ yum install python`

```

Comment:
@sachitanands I guess we no longer using python 2.x, we have moved to python 3.x, right ?